### PR TITLE
Feature/comments

### LINKED ISSUE #5: [3pt]  - Add comments counter - Student B
Even if the API gives you the number of comments you will create a specific function to calculate it.

Make sure that the correct number is displayed on the Comments pop up.

### LINKED ISSUE #8: [4pt]  - Create feature: add new comment - Student B
Implement the "add new comment" form on the Comments pop up according to the wireframe:

Screenshot 2021-06-27 at 22 32 36

When the user clicks on the "Comment" button, the data is recorded in the [Involvement API](https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270) and the screen is updated.

### LINKED ISSUE #11: [3pt]  - Display comments for a given item on the Comments pop-up - Student B
When the popup loads, the webapp retrieves data from the [Involvement API](https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270) to show the item comments.

Screenshot 2021-06-27 at 22 21 00

This task **does not** include the counter of the comments.

### PR DESCRIPTION
**Pull request by assadounto**
### I added 

- [ ] comment section

### CODE REVIEW

File: src/api/posts.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import Http from '../utils/http.js';
+import { API_INVOLVE_URL, APP_ID } from '../constants.js';
+
+const http = new Http(API_INVOLVE_URL);
+const endPoint = `apps/${APP_ID}/comments`;
+
+export default {
+  get(index) {
+    const endPoint2 = `apps/${APP_ID}/comments?item_id=${index}`;
+    return http.get(endPoint2);
```
Comment (teckim) [inline src/api/posts.js:10]: ## [SUGGESTION]
here you can put it like this:
```js
get(index) {
  return http.get(`${endPoint}?item_id=${index}`);
}
```

**Review by teckim — APPROVED** [review]
## Great job @assadounto

there's an optional suggestion in the comments.

Just a small note:
> I noticed that you updated my code, but since we are learning it's okay. But in a professional team, you will get yourself into trouble if you update someone's code without their permission.

**Review by teckim — APPROVED** [review]
## Great job @assadounto

there's an optional suggestion in the comments.

Just a small note:
> I noticed that you updated my code, but since we are learning it's okay. But in a professional team, you will get yourself into trouble if you update someone's code without their permission.

**Comment by assadounto** [comment]
Thank you very much for the notice I didn't realize that I changed ur code

**Pull request merged**